app/controllers/application_controller.rb

```diff
@@ -115,6 +116,12 @@ def swaddled
     response.headers["swaddled-by"] = "rubyforgood"
   end
 
+  def invalid_date
+    flash[:error] = "Date range not properly formatted."
+    params["filters"]["date_range"] = helpers.default_date
```

Comment:
Is this line necessary? Won't the default date range be selected after a redirect anyways?

Reply:
Thanks for catching that! Manually resetting the date_range is no longer necessary with our latest approach. Removed it [here](https://github.com/rubyforgood/human-essentials/pull/4925/commits/c5c15e2254926f922985cd0acd14feaf0e41ba63)